pages/common/firebase.md

```diff
@@ -0,0 +1,24 @@
+# firebase
+
+> Firebase Command Line Interface (CLI) Tools.
+> Used to test, manage, and deploy your Firebase project from the command line. More information: <https://www.npmjs.com/package/firebase-tools>.
+
+- Login:
+
+`firebase login`
+
+- Help:
+
+`firebase help`
+
+- List your firebase projects:
```

Comment:
Perhaps this will read better?
```suggestion
- List existing firebase projects:
```
We generally try to  avoid the 2nd person in tldr pages :-)